sympy/printing/printer.py

```diff
@@ -122,6 +122,32 @@ def print_my_latex(expr):
 
 Example of Custom Printing Method
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this example below, we want to print modulo operator in a different way in
+latex. Therefore, we override the method ``_latex`` of ``Mod``. See the example
+below::
+
+    from sympy import Symbol, Mod
+    from sympy.printing.latex import print_latex
+
+
+    class ModOp(Mod):
+        def _latex(self, printer=None):
+            a, b = [printer.doprint(i) for i in self.args]
+            return "\operatorname{Mod}{\left( %s,%s \\\\right)}" % (a,b)
```

Comment:
This needs to be 

```
return r"\\operatorname{Mod}{\\left( %s,%s \\\\right)}" % (a,b)
```

The `r` should always be used for latex strings. The extra backslashes are needed because the docstring itself is not a raw string (alternately, you could convert the docstring to a raw string). 

I would build the docs and double check that these latex examples render with the correct number of backslashes. Also, the next time you commit, the Travis tests will fail on this because we are now testing against invalid escapes like `\o`.